transforms/angle-brackets/transform.js

```diff
@@ -31,14 +31,24 @@ function isNestedComponentTagName(tagName) {
  *  Returns a capitalized tagname for angle brackets syntax
  *  {{my-component}} => MyComponent
  */
-function capitalizedTagName(tagname) {
-  return tagname
-    .split('-')
-    .map(s => {
-      if (!s) return '-';
-      return s[0].toUpperCase() + s.slice(1);
-    })
-    .join('');
+function capitalizedTagName(tagName) {
+  const SIMPLE_DASHERIZE_REGEXP = /[a-z]|\/|-/g;
+  const ALPHA = /[A-Za-z0-9]/;
+console.log(tagName)
+  tagName = tagName.replace(SIMPLE_DASHERIZE_REGEXP, (char, index) => {
+    if (char === '/') {
```

Comment:
It looks like this needs to folded in with the nested code or removed

Reply:
That's a great point. Since this change checks the character by character, we could achieve what `transformNestedTagName` is doing already in here. What are your thoughts on keeping this line in here and removing the `transformNestedTagName `  function completely and call only `capitalizedTagName ` in the `transformTagName` function, i.e. :

```
function transformTagName(tagName) {
  if (tagName.includes('.')) {
    return tagName;
  }
  return capitalizedTagName(tagName);
}
```
instead of 
```
function transformTagName(tagName) {
  if (tagName.includes('.')) {
    return tagName;
  }

  if (isNestedComponentTagName(tagName)) {
    return transformNestedTagName(tagName);
  }

  return capitalizedTagName(tagName);
}
```

Reply:
I think it’s great idea.

Reply:
We could actually move all of it.  Move the initial `.` check and then instead of two functions we can just have one function called `transformTagName`

Reply:
thats a wonderful idea